### PR TITLE
Update S3 Source to be EU compatible

### PR DESCRIPTION
**Pull request by mericsson**
### Proposed changes

Update documentation for S3 Source for EU regional customers on page https://segment.com/docs/connections/sources/catalog/cloud-apps/amazon-s3/

### Merge timing

ASAP once approved

### Related issues (optional)
https://segment.atlassian.net/browse/LIBWEB-1303

### CODE REVIEW

File: src/connections/sources/catalog/cloud-apps/amazon-s3/index.md [hunk 1]
```diff
@@ -92,6 +92,13 @@ var Analytics = require('analytics-node');
 var Objects = require('objects-node');
```
Comment (mericsson) [inline src/connections/sources/catalog/cloud-apps/amazon-s3/index.md:92]: Is there any updates we need to make here?

File: src/connections/sources/catalog/cloud-apps/amazon-s3/index.md [hunk 2]
```diff
@@ -92,6 +92,13 @@ var Analytics = require('analytics-node');
...
 
 var analytics = new Analytics(process.env.write_key);
+
+// NOTE: if you are sending to an EU based source, analytics will need to be initialized
```
Comment (mericsson) [inline src/connections/sources/catalog/cloud-apps/amazon-s3/index.md:96]:
```suggestion
// NOTE: if you are sending to an EU based source, `analytics-node` will need to be initialized
```

File: src/connections/sources/catalog/cloud-apps/amazon-s3/index.md [hunk 3]
```diff
@@ -92,6 +92,13 @@ var Analytics = require('analytics-node');
...
 
 var analytics = new Analytics(process.env.write_key);
+
+// NOTE: if you are sending to an EU based source, `analytics-node` will need to be initialized
+// with EU tracking API URL.
+// var analytics = new Analytics(process.env.write_key, {
+//   host: "https://events.eu1.segmentapis.com"
+// });
+
 var objects = new Objects(process.env.write_key);
```
Comment (mericsson) [inline src/connections/sources/catalog/cloud-apps/amazon-s3/index.md:102]:
```suggestion
var analytics = new Analytics(process.env.write_key);
var objects = new Objects(process.env.write_key);

// NOTE: if you are sending to an EU-based source, analytics and object SDKs
// need to be initialized with the appropriate host name.
// var analytics = new Analytics(process.env.write_key, {
//   host: "https://events.eu1.segmentapis.com"
// });
// var objects = new Objects(process.env.write_key, {
//   host: "http://objects.euw1.segment.com"
// });


```

File: src/connections/sources/catalog/cloud-apps/amazon-s3/index.md [hunk 4]
```diff
@@ -93,6 +93,17 @@ var Objects = require('objects-node');
...
+// NOTE: if you are sending to an EU-based source, analytics and object SDKs
+// need to be initialized with the appropriate host name.
+// var analytics = new Analytics(process.env.write_key, {
+//   host: "https://events.eu1.segmentapis.com"
+// });
+// var objects = new Objects(process.env.write_key, {
+//   host: "http://objects.euw1.segment.com"
+// });
+
+
```
Comment (mericsson) [inline src/connections/sources/catalog/cloud-apps/amazon-s3/index.md:106]:
```suggestion
```

File: src/connections/sources/catalog/cloud-apps/amazon-s3/index.md [hunk 5]
```diff
@@ -93,6 +93,16 @@ var Objects = require('objects-node');
 
 var analytics = new Analytics(process.env.write_key);
 var objects = new Objects(process.env.write_key);
+
+// NOTE: if you are sending to an EU-based source, analytics and object SDKs
```
Comment (mericsson) [inline src/connections/sources/catalog/cloud-apps/amazon-s3/index.md:97]:
```suggestion
// NOTE: if sending to an EU-based source, then the analytics and object SDKs
```

File: src/connections/sources/catalog/cloud-apps/amazon-s3/index.md [hunk 6]
```diff
@@ -93,6 +93,16 @@ var Objects = require('objects-node');
...
 var analytics = new Analytics(process.env.write_key);
 var objects = new Objects(process.env.write_key);
+
+// NOTE: if sending to an EU-based source, then the analytics and object SDKs
+// need to be initialized with the appropriate host name.
+// var analytics = new Analytics(process.env.write_key, {
+//   host: "https://events.eu1.segmentapis.com"
+// });
+// var objects = new Objects(process.env.write_key, {
+//   host: "http://objects.euw1.segment.com"
```
[the remark this answers is not in the document]
Comment (markzegarelli) [reply src/connections/sources/catalog/cloud-apps/amazon-s3/index.md:103]: @mericsson Should we ship these two together? Or are the ok to go out individually?

[the remark this answers is not in the document]
Comment (mericsson) [reply src/connections/sources/catalog/cloud-apps/amazon-s3/index.md:103]: OK to ship individually

**Review by markzegarelli — APPROVED** [review]

**Pull request merged**